packages/ods/src/components/datepicker/src/components/ods-datepicker/ods-datepicker.tsx

```diff
@@ -324,8 +330,17 @@ export class OdsDatepicker implements OdsFormElement {
     });
   }
 
+  private onInput(event: InputEvent): void {
+    const value = (event.target as HTMLInputElement).value;
+    if (!value || isDate(new Date(value))) {
```

Comment:
Why do we hide the clearable if the date is valid?
This causes issues when for example typing `11` on a clearable datepicker, button doesn't appear